src/processinfo.h

```diff
@@ -240,6 +216,29 @@ class ProcessInfo
     uint64_t _getcpu_offset;
     uint64_t _gettimeofday_offset;
     uint64_t _time_offset;
+
+    // Put <64-bit wide variabled here to ensure they are properly aligned.
```

Comment:
variabled -> variables

Reply:
Fixed

Reply:
Looks like you have merged the PR before pushing the typo fix. I guess you can take care of it in some other PR.